tests/requirements.txt

```diff
@@ -15,5 +15,6 @@ requests>=2.18.3
 
 # Optional Testing Dependencies
 nltk>=3.2
+# spacy>=2.0.18 
```

Comment:
Ok, I think this is a good choice - but if you create a follow on issue for extensions to this visualizer, could we add a note to it to revisit this? At that point in the future, hopefully we'll have seen some usage and we can evaluate whether spaCy is critical or not. Maybe we've also created more spaCy-specific visualizations as well or we can use that as an opportunity to review if there are any to be created! :-)